pkg/util/annotations/helpers.go

```diff
@@ -0,0 +1,48 @@
+// Package annotations implements annotation helper functions.
+package annotations
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+)
+
+// IsPaused returns true if the Cluster is paused or the object has the `paused` annotation.
+func IsPaused(o metav1.Object) bool {
+	return HasPausedAnnotation(o)
+}
+
+// HasPausedAnnotation returns true if the object has the `paused` annotation.
+func HasPausedAnnotation(o metav1.Object) bool {
+	return hasAnnotation(o, v1beta1.PausedAnnotation)
+}
+
+// AddAnnotations sets the desired annotations on the object and returns true if the annotations have changed.
+func AddAnnotations(o metav1.Object, desired map[string]string) bool {
+	if len(desired) == 0 {
+		return false
+	}
+	annotations := o.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+		o.SetAnnotations(annotations)
+	}
+	hasChanged := false
+	for k, v := range desired {
+		if cur, ok := annotations[k]; !ok || cur != v {
+			annotations[k] = v
+			hasChanged = true
+		}
+	}
+	return hasChanged
+}
```

Comment:
Is this used anywhere? Couldn't see it being used in the implementation

Reply:
I copied this package from upstream and removed everything unused. This one is used by the tests.

Reply:
The tests appear to be explicitly testing this function and not using any of the other functions we care about, right now I'd be tempted to delete those tests and this function as neither add anything from our use case as far as I can tell, do you agree?

Reply:
oh, I should look more carefully at the test code I copy. You're right. I remove the test file and the function